src/views/saphyr/SaphyrSamples.vue

```diff
@@ -155,7 +152,8 @@ export default {
     },
     async provider() {
       try {
-        this.items = await this.getMaterial(consts.MAT_TYPE_REQUESTS)
+        await this.setRequests()
+        this.items = this.requests
```

Comment:
We could possibly get rid of items in this component and call this.requests from the b-table, but looks like the clearPreFilter() method is using items. Although not sure how that works, if it does!?

Reply:
Removed pre filter and modified items.